docs/using_cameras.md

```diff
@@ -0,0 +1,95 @@
+
+# Using Cameras with ILLIXR
+
+ILLIXR supports a wide range of cameras, most of which are for the purpose of feeding images and IMU measurements to the system. <br>
+*To learn more about how it works, checkout [Building ILLIXR][12].* <br>
+*If you are interested in what topics these cameras feed into, checkout [ILLIXR plugins][10].*
+
+
+**Important Note**: <br>
+Before running any of these plugins below, it is important to comment out `offline_imu_cam` in `configs\native.yaml`. <br>
+
+## Add Calibration Parameters
+In order to add your camera's calibration extrinsics, you need to modify your choice of SLAM/VIO plugin. Follow these instructions to [modify a plugin][21].
+
+-   [`OpenVINS`][6]:
+    
+    Navigate to `ov_msckf/src`.<br>
+    Uncomment [this line][3] out in `slam2.cpp` in order to use ZED's calibration parameter for OpenVins. <br>
+    You can add your own calibration parameters in the same file.     
+
+## ZED Mini
+
+1. **Install ZED SDK** 
+
+    Install the latest version of the ZED SDK on [stereolabs.com][1].<br>
+    For more information, checkout the [ZED API documentation][2].
+
+2. **Get ZED's calibration parameters**
+
+    Both OpenVINS have a decent calirbation parameters for ZED. But if you wish to add your own: 
+    
+        /usr/local/zed/tools/ZED_Calibration
+
+    Your original factory calibration file is stored here
+
+        /usr/local/zed/settings/
+
+    Or download it from [calib.stereolabs.com][8].
+
+3. **Enable ZED in OpenVINS plugin**
+
+    This step is only required if using OpenVINS. Uncomment this [line](9) in the OpenVINS plugin. 
+
+4. **Run ILLIXR with ZED:**  
+
+    Uncomment `zed` in `configs/rt_slam_plugins.yaml` and run ILLIXR normally.
+
+## Intel Realsense 
+
+ILLIXR has been tested with Inteal RealSense D455, but it should work with any D or T series RealSense Camera. 
```

Comment:
OpenVINs parameter needs to be updated to work properly. Say that we will release the corresponding OpenVINs flag (#define realsense) soon.